use/developer/web/web-channel.md

```diff
@@ -0,0 +1,92 @@
+## About UCI :open_book:
+
+The Unified Communications Interface (UCI) aims to democratize the use of different communication channels such as WhatsApp, Telegram, SMS, email and more for governance use cases through a standard configurable manner that is reusable and scalable across all governance use cases.
```

Comment:
Is this consistent with the microsite?

Reply:
@ChakshuGautam what do you mean by microsite?